gpt4all-chat/modellist.cpp

```diff
@@ -495,7 +505,7 @@ const QList<QString> ModelList::userDefaultModelList() const
     for (ModelInfo *info : m_models) {
 
         // Only installed models that are meant for GUI are suitable as a default
-        if (!info->installed || info->disableGUI)
+        if (!info->installed || info->disableGUI || info->isEmbeddingModel)
```

Comment:
I'm fine with this, but these should be marked as disableGUI as well

Reply:
To me, disableGUI means "don't show this model in the GUI", not "don't show this model in the list of embedding models shown in the GUI". And an embedding model with disableGUI set to true shouldn't be shown in the GUI's list of embedding models.

Reply:
disableGUI means it is not a chat completion model essentially. probably could use better name ;)